server/apps/frontend/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "extreme-carpaccio-server-front",
```

So we have a `backend` and a `frontend`

```suggestion
  "name": "frontend",
```